### PR TITLE
Remove two unused includes in HdrHeap.h.

### PR DESCRIPTION
**Pull request by ywkaras**

### CODE REVIEW

**Comment by maskit** [comment]
I'll make a PR for this error soon. QPACK.h should include Arena.h.
```
In file included from Http3HeaderVIOAdaptor.h:26:0,
                 from Http3HeaderVIOAdaptor.cc:24:
QPACK.h:263:93: error: ���Arena��� has not been declared
   int _read_insert_with_name_ref(QUICStreamIO &stream_io, bool &is_static, uint16_t &index, Arena &arena, char **value,
                                                                                             ^~~~~
QPACK.h:265:62: error: ���Arena��� has not been declared
   int _read_insert_without_name_ref(QUICStreamIO &stream_io, Arena &arena, char **name, uint16_t &name_len, char **value,
                                                              ^~~~~
```

**Comment by maskit** [comment]
#6906 will fix the error.

**Comment by ywkaras** [comment]
[approve ci autest]

**Comment by ywkaras** [comment]
[approve ci autest]

File: proxy/http3/QPACK.h [hunk 1]
```diff
@@ -28,6 +28,7 @@
 #include "tscpp/util/IntrusiveDList.h"
 #include "MIME.h"
 #include "QUICApplication.h"
+#include <tscore/Arena.h>
```
Comment (randall) [inline proxy/http3/QPACK.h:31]: shouldn't this be `"` like the rest of the includes preceding it?

Comment (ywkaras) [reply proxy/http3/QPACK.h:31]: My opinion is that it's more clear to use <> not "" when the included file is not in the same directory as the including file (or is not specified with an absolute path).

**Comment by maskit** [comment]
Why not merge #6906 first and make this one purely removing the unused includes? The PR is labeled QUIC but it's for master.

**Comment by ywkaras** [comment]
OK, I will mark this as draft until 6906 is merged.

**Pull request merged**